drake/doc/homebrew.rst

```diff
@@ -1,21 +1,20 @@
-********
-Mac (using homebrew -- recommended for mac)
-********
+*********************************************
+OS X (using Homebrew -- recommended for OS X)
+*********************************************
 
 We have provided some scripts to simplify the installation of prerequisites.  However, we do assume that you have already installed
 
 * XQuartz / X11
 * XCode and the command line tools (``xcode-select --install``)
 
-Now you can install the prerequistes::
+You can install the prerequisites using the ``install_prereqs.sh`` script::
 
 	brew update
-	brew install cmake
+	brew upgrade  # optional
 	cd drake-distro
-	make options # use the gui to choose which externals you want, then press c to configure, then g to generate makefiles and exit
-	make download-all
 	./install_prereqs.sh homebrew

```

Comment:
I think we should go all of the way with this PR and remove the install_prereqs files completely.  We should simply copy the working lines in install prereqs into the instructions here.   my understanding is that this is the standard workflow that people expect, and provides extra visibility.